### PR TITLE
Add forward and backward overlap icons

### PR DESCRIPTION
**Pull request by james-yeoman**
I'm not an artist in the slightest, but by using `copy` as a base, I was able to get these icons looking serviceable.

These icons may need to be measured in something more capable, since I used [svgviewer](https://www.svgviewer.dev), which doesn't have pixel measurement capabilities. It's also not the best for dimensional accuracy, as you can see from this screenshot.

<details>
<summary>Collapsible screenshot and svg code from svgviewer</summary>

image

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-bring-to-front">
  <!--  Debug Border, remove before SVGOMG  -->
  <rect width="24" height="24" x="0" y="0" stroke-width="1" stroke="red" />
  
  <rect width="8" height="8" x="8" y="8" rx="2" ry="2"/>
  <path d="M 4 10 c -1 0 -2 -1 -2 -2 v -4 c 0 -1 1 -2 2 -2 h 4 c 1 0 2 1 2 2"/>
  <path d="M 14 20 c 0 1 1 2 2 2 h 4 c 1 0 2 -1 2 -2 v -4 c 0 -1 -1 -2 -2 -2"/>
</svg>

<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-send-to-back">
  <!--  Debug Border, remove before SVGOMG  -->
  <rect width="24" height="24" x="0" y="0" stroke-width="1" stroke="red" />
  
  <rect width="8" height="8" x="2" y="2" rx="2" ry="2" />
  <rect width="8" height="8" x="14" y="14" rx="2" ry="2" />
  <path d="M 7 14 v 1 c 0 1 1 2 2 2 h 1" />
  <path d="M 14 7 h 1 c 1 0 2 1 2 2 v 1" />
</svg>


<!-- Lucide copy icon for reference -->
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-copy">
  <!--  Debug Border  -->
  <rect width="24" height="24" x="0" y="0" stroke-width="1" stroke="red" />
  
  <rect width="14" height="14" x="8" y="8" rx="2" ry="2"/>
  <path d="M 4 16 c -1.1 0 -2 -.9 -2 -2 V 4 c 0 -1.1 .9 -2 2 -2 h 10 c 1.1 0 2 .9 2 2"/>
</svg>

```
</details>

When eyeballing the icons, the spacing looks the same as the `copy` icon, and so should be design guide compliant. That said, can someone please verify in something like Illustrator or another capable program?

### CODE REVIEW

**Comment by james-yeoman** [comment]
Ooh, I didn't know about that workflow. The icon x-rays confirm that the spacing is within the design guide!

File: icons/bring-to-front.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="8" height="8" x="8" y="8" rx="2" ry="2" />
+  <path d="M4 10c-1 0-2-1-2-2V4c0-1 1-2 2-2h4c1 0 2 1 2 2m4 16c0 1 1 2 2 2h4c1 0 2-1 2-2v-4c0-1-1-2-2-2" />
```
Comment (jguddas) [inline icons/bring-to-front.svg:13]:
```suggestion
  <rect x="8" y="8" width="8" height="8" rx="2" />
  <path d="M4 10a2 2 0 0 1-2-2V4a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2" />
  <path d="M14 20a2 2 0 0 0 2 2h4a2 2 0 0 0 2-2v-4a2 2 0 0 0-2-2" />
```

File: icons/send-to-back.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="8" height="8" x="2" y="2" rx="2" ry="2" />
+  <rect width="8" height="8" x="14" y="14" rx="2" ry="2" />
+  <path d="M7 14v1c0 1 1 2 2 2h1m4-10h1c1 0 2 1 2 2v1" />
```
Comment (jguddas) [inline icons/send-to-back.svg:14]:
```suggestion
  <rect x="14" y="14" width="8" height="8" rx="2" />
  <rect x="2" y="2" width="8" height="8" rx="2" />
  <path d="M7 14v1a2 2 0 0 0 2 2h1" />
  <path d="M14 7h1a2 2 0 0 1 2 2v1" />
```

**Review by jguddas — CHANGES_REQUESTED** [review]
I fixed some things using https://lucide-site-git-feat-added-edit-page-jguddas.vercel.app/edit .

File: icons/send-to-back.svg (hunk 2 above)
Comment (james-yeoman) [reply icons/send-to-back.svg:14]: ditto

**Comment by james-yeoman** [comment]
Is there anything left that needs tweaking? Or is this just waiting on a release candidate?

File: icons/bring-to-front.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "james-yeoman"
+  ],
```
Comment (jguddas) [inline icons/bring-to-front.json:5]:
```suggestion
  "contributors": [
    "james-yeoman",
    "jguddas"
  ],
```

File: icons/bring-to-front.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "tags": [
+    "bring to front",
+    "bring forward"
+  ],
```
Comment (jguddas) [inline icons/bring-to-front.json:9]:
```suggestion
  "tags": [
    "bring",
    "send",
    "move",
    "over",
    "forward",
    "front",
    "layer"
  ],
```

File: icons/send-to-back.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "james-yeoman"
+  ],
```
Comment (jguddas) [inline icons/send-to-back.json:5]:
```suggestion
  "contributors": [
    "james-yeoman",
    "jguddas"
  ],
```

File: icons/send-to-back.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  "tags": [
+    "send to back",
+    "send backward"
+  ],
```
Comment (jguddas) [inline icons/send-to-back.json:9]:
```suggestion
  "tags": [
    "bring",
    "send",
    "move",
    "over",
    "back",
    "backwards",
    "layer"
  ],
```

**Review by jguddas — COMMENTED** [review]
I think the icons are good 👍 

@karsa-mistmere, @ericfennis what do you think?

**Review by jguddas — APPROVED** [review]

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]
Nice work @james-yeoman !!

**Pull request merged**